README.md

```diff
@@ -302,6 +302,25 @@ target_link_libraries(${MY_TARGET_NAME} argh)
 
 ```
 
+## Tipi.build
+
+`argh` can be easily used in [tipi.build](https://tipi.build) projects simply by adding the following entry to your `.tipi/deps`:
+
+```json
+{
+    "adishavit/argh": { "x": [ "*.cpp" ]}
+}
+```
+
+> `argh` itself is a header-only library hence the exclusion of all `.cpp` files above. This isn't stricly required to inlcude `argh` but speeds up the build.
```

Comment:
In tipi we use a source scan based approach to dependency management, instead of having you provide some type of script to make `argh` usable in someone else's project. Basically when someone adds that line to *his* `.tipi/deps` files, tipi will pull `argh` straight from Github and install it in a project specific sysroot.

By excluding the *.cpp files that are not actually part of `argh` (but rather only test cases and examples) these files are not being scanned or compiled which can save a bit of time.

Reply:
I don't completely understand.

So is adding that json snippet with the `*.cpp` needed or not?
Is there some way or a need to exclude stuff?